source/GameMode.cpp

```diff
@@ -233,6 +241,8 @@ void GameMode::update(float timestep) {
 		if (!donutModel->isJumping() && playerID != breach->getPlayer() &&
 			diff < globals::BREACH_WIDTH && breach->getHealth() != 0) {
 			// Slow player by drag factor
+			CULog("Player %d at angle %f slowed by breach at %f with id %d", playerID,
+				  donutModel->getAngle(), breach->getAngle(), breach->getPlayer());
```

Comment:
This seems like a potentially high frequency log. Do we want that in here?

Reply:
ah, my bad. should've been deleted